build/gulpfile.reh.js

```diff
@@ -63,6 +63,8 @@ const BUILD_TARGETS = [
 const serverResourceIncludes = [
 	// --- Start Positron ---
 	'out-build/react-dom/client.js',
+	'out-build/vs/code/browser/workbench/rsLoginCheck.js',
+	'out-build/vs/code/browser/workbench/urlPorts.js',
```

Comment:
Out of curiosity, why don't these entries exist upstream in our vscode-server fork?

Reply:
```suggestion
```

Just tested a local build without this and it's all good, the UrlPorts code ends up in the workbench.js megafile. Seems like just a lingering line from figuring out which sources were missing from the build.

Reply:
Maybe we want to add `rsLoginCheck.js` to `serverResourceIncludes` in upstream instead, and remove the copy step from `upload-package.sh`? From the history of the upload-package.sh file, it seems like the copy step may have been added there since it was missing in the binary, which this change should cover. 

I'll open an issue in upstream to consider this change. If we move it over now, we'll need to test vscode in workbench and merge upstream here, which we may not want to focus on at this moment.